frontend/packages/yki/src/redux/selectors/examSessions.ts

```diff
@@ -42,6 +42,7 @@ const filterExamSessions = (
       (es) =>
         es.participants < es.max_participants ||
```

Comment:
Tässä varmaan pitäis myös tsekata että varsinainen ilmoittautuminen on myös auki eikä mahd. jälki-ilmoittautuminen (tai jälki-ilmo päättynyt) ? Täällä olevan filter-ehdon vois kai myös korvata ihan vaan tuolla `hasRoom` tsekkauksella?